tracker/models.py

```diff
@@ -121,6 +121,12 @@ def is_staff(self) -> models.BooleanField:
         :return: bool
         """
         return self.is_admin
+    
+    def is_project_lead(self):
```

Comment:
Pls add return type and update your docstring

Reply:
addressed in [c3d18b1](https://github.com/KostCode-Organization/hackathon_bot/pull/44/commits/c3d18b1846618fec4417236ec21a94fb4237172a)